### PR TITLE
Adding monoHiggs_2HDMa datacards to mg265UL

### PR DESCRIPTION
**Pull request by sslomte**
Using monoHiggs 2HDMa datacards from pre-legacy, and updated LHAPDF ID to 325300 as per UL recommendation. These datacards will be used for central production of Run-II UL MC samples for monoHiggs bbbar analysis.

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/monoHiggs_2HDMa/ggProduction/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+     0.0     = polbeam1 ! beam polarization for beam 1
+     0.0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+     lhapdf    = pdlabel     ! PDF set
+     325300    = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/monoHiggs_2HDMa/ggProduction/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY_run_card.dat:48]: "Pseudoscalar_2HDM" is 4f, you should use 325500 or use:
$DEFAULT_PDF_SETS    = lhaid    
 $DEFAULT_PDF_MEMBERS    = reweight_PDF  ! if pdlabel=lhapdf, this is the lhapdf number

File: bin/MadGraph5_aMCatNLO/cards/monoHiggs_2HDMa/ggProduction/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+#***********************************************************************
+ -1.0  =  ktdurham
+ 0.4   =  dparameter
+ -1.0  =  ptlund
+ 1, 2, 3, 4, 5, 6, 21, 82  =  pdgs_for_merging_cut ! PDGs for two cuts above
+#*********************************************************************
+# maximal pdg code for quark to be considered as a light jet         *
+# (otherwise b cuts are applied)                                     *
+#*********************************************************************
+ 5 = maxjetflavor    ! Maximum jet pdg code
```
Comment (menglu21) [inline bin/MadGraph5_aMCatNLO/cards/monoHiggs_2HDMa/ggProduction/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY/2HDMa_gg_sinp_theta_tanb_beta_mXd_10_MH3_YY_MH4_XX_MH2_YY_MHC_YY_run_card.dat:261]: use 4 here

**Review by menglu21 — COMMENTED** [review]
besides the comments inline, please:
- move all the cards to 2017/13TeV directory
- remove the run_card.dat~

**Comment by sslomte** [comment]
Thank you, @menglu21 for the prompt response. Suggested edits are in place now.

**Pull request merged**